lib/beaker-hostgenerator/data.rb

```diff
@@ -339,6 +339,11 @@ def osinfo
             'template' => 'fedora-24-x86_64'
           }
         },
+        'huaweios6-POWER' => {
+          :general => {
+            'platform' => 'huaweios-6-power'
```

Comment:
Oops, good catch, forgot the `pc` bit at the end.

Judging from your other comment, it sounds like these are two distinct architectures that need to distinguished further.  I'll update the `--list` output accordingly.